### PR TITLE
Backports and release notes for v3.4.0-RC1

### PR DESCRIPTION
**Pull request by mheon**
As the title says.

Let's hope CI is happy.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,84 @@
...
+- Support for init containers has also been added to `podman play kube` and `podman generate kube` - init containers contained in Kubernetes YAML will be created as Podman init containers, and YAML generated by Podman will include any init containers created.
+- The `podman play kube` command now supports building images. If the `--build` option is given and a directory with the name of the specified image exists in the current working directory and contains a valid Containerfile or Dockerfile, the image will be built and used for the container.
+- The `podman play kube` command now supports a new option, `--teardown`, which removes any pods and containers created by the given Kubernetes YAML.
+- A new command has been added, `podman pod logs`, to return logs for all containers in a pod at the same time.
+- Two new commands have been added, `podman volume export` (to export a volume to a tar file) and `podman volume import) (to populate a volume from a given tar file).
```
Comment (Luap99) [inline RELEASE_NOTES.md:10]:
```suggestion
- Two new commands have been added, `podman volume export` (to export a volume to a tar file) and `podman volume import`) (to populate a volume from a given tar file).
```

**Comment by mheon** [comment]
@baude @TomSweeneyRedHat PTAL at the release notes please

**Comment by mheon** [comment]
The build each commit job appears to be busted, it's building commits from last year.

**Comment by mheon** [comment]
Build each commit now complaining about a commit that doesn't exist in the tree...

**Comment by edsantiago** [comment]
I can reproduce the `build-each-commit` error on my laptop. It looks like fb564d88b6a84945255bcf90df74a88240ac370a is broken, with the fix coming in 3 or so commits later (sorry, `git rebase -x` doesn't show me the SHA that fixes it.

**Comment by mheon** [comment]
So it's just reporting the wrong commit hashes? Lovely...

**Comment by edsantiago** [comment]
I don't understand your comment. `git lp` shows 29 commits, and the one I reported is one of those 29.

**Comment by Luap99** [comment]
It looks like you are missing the --ssh option in the podman build page. https://github.com/containers/podman/blob/main/docs/source/markdown/podman-build.1.md#--sshdefaultidsocket

**Comment by mheon** [comment]
@edsantiago Cirrus' build-each-commit job was giving me SHAs that don't exist in the tree - so perhaps it works locally, but it decidedly does not on upstream CI.

**Comment by mheon** [comment]
I think CI is starting to pass.

**Comment by cevich** [comment]
The `.cirrus.yml` changes LGTM

**Comment by mheon** [comment]
Buildah tests failing - probably need to copy those from main as well.

**Review by Luap99 — COMMENTED** [review]
LGTM

**Review by lsm5 — APPROVED** [review]
LGTM

**Pull request merged**